it/mail/chrome/messenger/messenger.properties

```diff
@@ -791,45 +791,45 @@ trialUpgradeButton.label=Acquista Postbox...
 trialContinueButton.label=Non in linea
 trialEditButton.label=Inserisci licenza
 trialValidateInfo1.label=Potrai ancora eseguire Postbox offline e visualizzare o esportare i tuoi dati.
-trialValidateInfo2.label=Non potrai eseguire Postbox in modalità online finché non inserirai un codice di licenza valido. Visita il nostro negozio Web sicuro cliccando sul pulsante "Purchase" qui sotto.
+trialValidateInfo2.label=Non potrai eseguire Postbox in modalità online finché non inserirai un codice di licenza valido. Visita il nostro negozio Web sicuro cliccando sul pulsante "Acquista" qui sotto.
 
 invalidExpiringDaysNotification.title=Il tuo codice di licenza non è valido e questa versione demo scadrà tra %1$S giorni.
 invalidExpiringDayNotification.title=Il tuo codice di licenza non è valido e questa versione demo scadrà tra %1$S giorno.
 invalidExpiringTodayNotification.title=Il tuo codice di licenza non è valido e questa versione demo scade oggi.
 invalidExpiredNotification.title=Codice licenza non valido. Effettua la registrazione o visita il nostro negozio online per acquistare Postbox.
 invalidExpiredNotification2.title=Il tuo codice di licenza non è valido.
-invalidExpiringNotificationUpgrade.label=Purchase Postbox...
+invalidExpiringNotificationUpgrade.label=Acquista Postbox...
 invalidExpiringNotificationUpgrade.accessKey=P
 invalidExpiringNotificationContact.label=Contatta Postbox...
 invalidExpiringNotificationContact.accessKey=C
 invalidWindow.title=Codice licenza non valido
 invalidUpgradeButton.label=Acquista Postbox...
 invalidContinueButton.label=Modalità offline
-invalidEditButton.label=Inserire llicenza
-invalidValidateInfo1.label=Postbox ha rilevato un codice di licenza non valido. Se hai ricevuto questo messaggio per errore, contattaci al
-invalidValidateInfo1a.label=Postbox Support
+invalidEditButton.label=Inserire licenza
+invalidValidateInfo1.label=Postbox ha rilevato un codice di licenza non valido. Se hai ricevuto questo messaggio per errore, contatta l'
+invalidValidateInfo1a.label=Assistenza Postbox
```

Comment:
CORRECTION NEEDED. 

I would replace "Assistenza Postbox" with "Supporto Postbox" in order to avoid an unnecessary repetition (the word "assistenza" appears on line 811 too).